test/Faker/Provider/en_US/PhoneNumberTest.php

```diff
@@ -2,84 +2,75 @@
 
 namespace Faker\Test\Provider\en_US;
 
-use Faker\Generator;
 use Faker\Provider\en_US\PhoneNumber;
 use Faker\Test\TestCase;
 
 final class PhoneNumberTest extends TestCase
 {
-
-    /**
-     * @var Generator
-     */
-    private $faker;
-
-    protected function setUp(): void
-    {
-        $faker = new Generator();
-        $faker->addProvider(new PhoneNumber($faker));
-        $this->faker = $faker;
-    }
-
     public function testPhoneNumber()
     {
-        for ($i = 0; $i < 100; $i++) {
+        for ($i = 0; $i < 100; ++$i) {
```

Comment:
We could remove all of these for loops that test different randomized versions when we're seeding the number generator I guess?